server.py

```diff
@@ -2,14 +2,10 @@
 from transform import app
 import logging
 import logging.handlers
-import os
 
 
 if __name__ == '__main__':
     # Startup
     logging.basicConfig(level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
     handler = logging.handlers.RotatingFileHandler(settings.LOGGING_LOCATION, maxBytes=20000, backupCount=5)
-    handler.setFormatter(logging.Formatter(settings.LOGGING_FORMAT))
-    app.logger.addHandler(handler)
-    port = int(os.getenv("PORT"))
-    app.run(debug=True, host='0.0.0.0', port=port)
```

Comment:
Sorry, doing this in lots of places and replaced with similar code from another service.